web/client/translations/data.it-IT.json

```
@@ -1483,7 +1483,8 @@
             },
             "format": {
                 "noOption": "Nessuna opzione",
-                "loading": "Caricamento in corso..."
+                "loading": "Caricamento in corso...",
+                "refresh": "Recupera il formato supportato"
```

I think it should be plural also on the other translations.
```suggestion
                "refresh": "Recupera i formati supportati"
```